src/clients/client.ts

```diff
@@ -29,6 +29,7 @@ import {
   KeysResults,
   TasksResults,
   EnqueuedTaskObject,
+  IndexesSwapParams,
```

Comment:
```suggestion
  SwapIndexesParams,
```
SwapIndexesParams is better since the route name is `swap-indexes` don't you think?